engine/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateService.java

```diff
@@ -22,6 +26,7 @@ public String renderTemplate(User user, Action action, TemplateInstance template
         return templateInstance
                 .data("action", action)
                 .data("user", user)
+                .data("config", config)
```

Comment:
`config` is a bit of a generic term that does not really carry the meaning of the available features behind it. I wonder if we could find something more "precise".

Reply:
I wonder that too - other option could be just: `helpers`

Reply:
WDYT of simply using `environment`? The methods could then be named `environment#name` and `environment#absolueUrl`?

Reply:
Changed to `environment` but still prefering the short `environment.url`